app/src/main/java/org/simple/clinic/teleconsultlog/prescription/TeleconsultPrescriptionEvent.kt

```diff
@@ -21,3 +21,7 @@ data class DataForNextClickLoaded(
 data class NextButtonClicked(val medicalInstructions: String, val medicalRegistrationId: String) : TeleconsultPrescriptionEvent() {
   override val analyticsName: String = "Teleconsult Prescription:Next Clicked"
 }
+
+data class PrescriptionCreated(
+    val medicalInstructions: String
```

Comment:
What do you think of tracking this in `TeleconsultPrescriptionModel` so we don't need to forward this via an event?

Reply:
That would be the only use case for it, like we won't be using it for anything else. But yeah, I can move it to model.

Reply:
No need to do it in this PR, we can do it later if needed.